web/src/providers/SettingsValues/SettingsValues.provider.tsx

```diff
@@ -66,6 +75,17 @@ const DataStoreConfigProvider = ({children}: IProps) => {
     );
   }, [dispatch]);
 
+  // Config
+  const {data: config = Config()} = useGetConfigQuery({});
+
+  useEffect(() => {
```

Comment:
I like this, but let's make sure we don't forget this provider and this side effect is the one handling the analytics switch